features/webxr-hit-test.yml

```
@@ -0,0 +1,23 @@
+name: WebXR Hit Test
+description: WebXR Hit Test allows to place objects in the real world by casting rays that can be evaluated against the underlying platform's understanding of the real world geometry.
```

I'm not sure what "allows to place" means. Thinking this might be a good direction to go in (assuming I'm getting the gist here):

```suggestion
description: The `getHitTestResults()` method of an `XRFrame` object helps place objects in an augmented reality session by finding where cast rays intersect with real-world geometry.
```